content/2021/russian/_index.md

```diff
@@ -10,13 +10,13 @@ headline: "JakartaOne Livestream - Russian"
 tagline: "March 31, 2021"
 tagline_suffix: "<p><img width='180' class='img-responsive center-block' src='/images/jakarta/jakarta-ee-logo.svg' alt='Jakarta EE: The New Home of Cloud Native Java'></p>"
 layout: "single"
-menu_name: "russia"
+menu_name: "russian"
 aliases:
-  - /russia2021/
+  - /russian2021/
```

Comment:
We should create another alias for the previous page to avoid breaking any existing links.

Reply:
Actually, it does not hurt to keep /russia2021/ in the list just in case.

Reply:
Updated